Gemfile

```diff
@@ -17,4 +17,5 @@ end
 
 group :development, :test do
   gem "simplecov"
+  gem "debug", platforms: %i[mri mingw x64_mingw]
```

Comment:
You might have forgotten to remove this after debugging!

Reply:
I thought it a valuable addition. Of course, can remove if unwanted by the maintainers.